### PR TITLE
Paramedic helmet fix and "mute" feature

### PR DESCRIPTION
**Pull request by SirRichardFrancis**
## About The Pull Request

Fixed an issue with synthetic heart triggering suit sensors.
Added mute/unmute button to paramedic tablet's interface, allowing to disable paramedic helmet's notifications for specific people.

image

image

## Why It's Good For The Game

Fixes cool.
QoL features cool too.

## Changelog
:cl:
add: mute/unmute button to paramedic tablet's interface
fix: synthetic heart no longer trigger suit sensors
/:cl:

### CODE REVIEW

**Comment by MLGTASTICa** [comment]
Why should paramedics be able to mute it, its kind of counterintuitive to their job and seems to lean more onto the powergaming side of things. If they are using that armor they need to be constantly told whenever someone is dying..

**Comment by Kegdo** [comment]
synthetic hearts and spaced dead/unretrievable people say hello.

**Comment by Wouju** [comment]
> Why should paramedics be able to mute it, its kind of counterintuitive to their job and seems to lean more onto the powergaming side of things. If they are using that armor they need to be constantly told whenever someone is dying..

The mute function is to „mute” specific people from what i understood. How the fuck is that powergaming? If you have some idiot IH getting themselves shot every 3 minutes eventually you will give up hope and „ignore” them (read: not notice them)

**Comment by SirRichardFrancis** [comment]
> Why should paramedics be able to mute it, its kind of counterintuitive to their job and seems to lean more onto the powergaming side of things. If they are using that armor they need to be constantly told whenever someone is dying..

To be honest, if you're planning to powergame, step one would be finding a better helmet.
Plus, paramed helmet have option to disable all notifications already.
This addition for muting individual people, bodies are hard to recover sometimes and paramedic may simply don't give a fuck about vagabonds fighting roaches all the time.

**Comment by Jan-Blasiak** [comment]
And/or people who died ages ago in space and still clutter the interface.

File: code/datums/repositories/crew/vital.dm [hunk 1]
```diff
@@ -24,7 +24,7 @@
 				if(PULSE_THREADY)
 					crew_data["alert"] = TRUE
 					crew_data["pulse_span"] = "bad"
-		else if(BP_IS_ROBOTIC(O))
+		else if(O)
```
Comment (TheShown911) [inline code/datums/repositories/crew/vital.dm:27]: No need to check for O twice, you can put one if(O) and these 2 tabbed under it.

File: code/__HELPERS/global_lists.dm [hunk 2]
```diff
@@ -135,6 +135,9 @@ var/global/list/syndicate_access = list(access_maint_tunnels, access_syndicate,
 //A list of slots where an item doesn't count as "worn" if it's in one of them
 var/global/list/unworn_slots = list(slot_l_hand,slot_r_hand, slot_l_store, slot_r_store,slot_robot_equip_1,slot_robot_equip_2,slot_robot_equip_3)
 
+//Names that shouldn't trigger notifications about low health
+GLOBAL_LIST_EMPTY(ignore_health_alerts_from)
```
[the remark this answers is not in the document]
Comment (SirRichardFrancis) [reply code/__HELPERS/global_lists.dm:139]: Yes. Not perfect, but lesser evil than linking tablets to helmets or some other overcomplication.

[the remark this answers is not in the document]
Comment (drexample) [reply code/__HELPERS/global_lists.dm:139]: BRUH

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**